src/Components/Planner.js

```diff
@@ -0,0 +1,118 @@
+// React Hooks
+import { useState, useEffect } from "react";
+// Firebase
+import { getDatabase, ref, onValue, push, remove, set, update } from "firebase/database";
+import firebase from "../firebase";
+// Forms Import
+import Forms from "./Forms";
+
+// Planner Section
+const Planner = () => {
+    const [exercises, setExercises] = useState([]);
+    const [userDayInput, setUserDayInput] = useState("");
+    const [userTypeInput, setUserTypeInput] = useState("");
+    const [userDurationInput, setUserDurationInput] = useState("");
+    const [error, setError] = useState(false);
+    // Firebase Database
+    useEffect(() => {
+        const database = getDatabase(firebase)
+        const dbRef = ref(database)
+
+        onValue(dbRef, (res) => {
+            const newState = [];
+            const data = res.val();
+
+            for (let key in data) {
+                newState.push({
+                    key: key,
+                    day: data[key].day,
+                    type: data[key].type,
+                    duration: data[key].duration
+                });
+            }
+
+            setExercises(newState);
+        })
+    }, [])
+    // User Database
+    const newExercise = {
+        day: userDayInput,
+        type: userTypeInput,
+        duration: userDurationInput
+    }
+    // User's Values
+    const handleDayChange = (e) => {
+        setUserDayInput(e.target.value);
+    }
+    const handleTypeChange = (e) => {
+        setUserTypeInput(e.target.value);
+    }
+    const handleDurationChange = (e) => {
+        setUserDurationInput(e.target.value);
+    }
+    // Submit Request
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        const database = getDatabase(firebase);
+        const dbRef = ref(database);
+
+        const displayErr = (value) => {
```

Comment:
Great job with error handling! Maybe next time also add in an Error page that will display when a page isn't found?